src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -1090,10 +1090,24 @@ private static void ReportImplicitImplementationMatchDiagnostics(Symbol interfac
                     else if (MemberSignatureComparer.RuntimeImplicitImplementationComparer.Equals(interfaceMember, member) && !member.IsAccessor())
                     {
                         // CONSIDER: Dev10 does not seem to report this for indexers or their accessors.
-                        diagnostics.Add(ErrorCode.WRN_MultipleRuntimeImplementationMatches, member.Locations[0], member, interfaceMember, implementingType);
+                        diagnostics.Add(ErrorCode.WRN_MultipleRuntimeImplementationMatches, GetDiagnosticLocation(member), member, interfaceMember, implementingType);
                     }
                 }
             }
+
+            Location GetDiagnosticLocation(Symbol member)
```

Comment:
>Symbol member [](start = 43, length = 13)

It feels strange that we are passing a ```member```, but not passing ```implementingType``` and ```interfaceMember``` . Consider either passing all, or neither.

Reply:
We are only passing a `member` because `WRN_MultipleRuntimeImplementationMatches` may be reported in a location different from `implicitImpl`.